src/python/pants/backend/project_info/tasks/idea_plugin_gen.py

```diff
@@ -63,6 +63,8 @@ def register_options(cls, register):
     # scala/java-language level should use what Pants already knows.
     register('--open', type=bool, default=True,
              help='Attempts to open the generated project in IDEA.')
+    register('--incremental-import', type=int, default=None,
+             help='Enable incremental import of targets with the given graph depth.')
```

Comment:
I wasn't sure about specifying depth number directly, as opposed to a boolean, because currently there's no way to specify depth prior to `pants export` https://github.com/pantsbuild/intellij-pants-plugin/blob/43ec02d80c1a87291c6533ea35997cd82afbba88/src/com/twitter/intellij/pants/components/impl/PantsProjectComponentImpl.java#L136

whereas depth number is now collected via GUI after `pants export` https://github.com/pantsbuild/intellij-pants-plugin/blob/3027a708a87f993590354383d90a1f6872e92dd3/src/com/twitter/intellij/pants/service/project/resolver/PantsCreateModulesExtension.java#L93-L115 (yes a little awkward there)

That said, it's good to set this up first, then get the awkwardness fixed afterwards, but do you think  it would be a good idea to clarify in the help msg?
```
Enable incremental import of targets with the given graph depth. 
Depending on the version of IntelliJ Pants plugin, depth info may be prompted again or ignored.
```